jira/client.py

```diff
@@ -196,7 +197,7 @@ class JIRA(object):
     AGILE_BASE_URL = GreenHopperResource.AGILE_BASE_URL
 
     def __init__(self, server=None, options=None, basic_auth=None, oauth=None, jwt=None, kerberos=False,
-                 validate=False, get_server_info=True, async=False, logging=True, max_retries=3, proxies=None):
+                 validate=False, get_server_info=True, async=False, max_retries=3, proxies=None):
         """Construct a JIRA client instance.
```

Comment:
@asqui it seems that you missed to observe that `JIRA.__init__()` constructor accepts the logger parameter. If we are to remove it we need to be careful in order not to break code that is already using it. When I mean preventing breaking, I mean to keep it running instead of getting a compilation error because we removed the parameter.